vendor/arm/pmu/pmu_profiler.cpp

```diff
@@ -103,19 +101,19 @@ const CpuMeasurements &PmuProfiler::sample()
 			continue;
 		}
 
-		try
+		auto value = pmu_counter->second.get_value<long long>();
+		if (value < 0)
+		{
+			log(LogSeverity::Warn, "Failed to get value from PMU: {}.", std::strerror(errno));
```

Comment:
I attempted to unify types used within the project with `hwcpipe::IntType` and `hwcpipe::DoubleType`. I also added a validity check to Value so that other projects that use hwcpipe can verify if the counter was obtained correctly